browser/resources/settings/brave_default_extensions_page/change_ipfs_gateway_dialog.js

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2020 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+/**
+ * @fileoverview
+ * 'change-ipfs-gateway-dialog' provides a dialog to configure the public
+ * IPFS gateway address.
+ */
+import '../settings_shared_css.m.js';
+
+import 'chrome://resources/cr_elements/cr_button/cr_button.m.js';
+import 'chrome://resources/cr_elements/cr_dialog/cr_dialog.m.js';
+import 'chrome://resources/cr_elements/cr_input/cr_input.m.js';
+
+import {BraveDefaultExtensionsBrowserProxyImpl} from './brave_default_extensions_browser_proxy.m.js';
+import {Polymer, html} from 'chrome://resources/polymer/v3_0/polymer/polymer_bundled.min.js';
+import {I18nBehavior} from 'chrome://resources/js/i18n_behavior.m.js';
```

Comment:
nit: prefer import paths sorted by path hierarchy, so: global paths then ancestor paths then local paths.

Reply:
Fixed in 6614869913a0157aba7701a13ab1ae73c4d3e6e6